### ISSUE TITLE
[COMP] `aria-current` styles should be in the `a` tag not `navListItem`

### ISSUE THREAD

**Issue by marlonmarcello** [wethegit/component-library#87]
Reference file
https://github.com/wethegit/component-library/blob/main/packages/wethegit-components/src/components/navigation/components/nav-list/nav-list.module.scss#L19
